scripts/exgdas_atmos_vminmon.sh

```diff
@@ -0,0 +1,119 @@
+#! /usr/bin/env bash
+
+source "$HOMEgfs/ush/preamble.sh"
```

Comment:
Same comment, remove the preamble line but can most likely keep the rest of the v17 updates in this script.